eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -309,36 +309,33 @@ stages:
     ### Job for ultralite build ###
     - template: ../jobs/vmr-build.yml
       parameters:
-        buildName: MarinerCrossX64
+        buildName: Ubuntu2204
         isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
         vmrBranch: ${{ variables.VmrBranch }}
         architecture: x64
         pool:
           name: ${{ variables.defaultPoolName }}
           demands: ${{ variables.defaultPoolDemandsLinux }}
-        container: ${{ variables.marinerX64CrossContainer }}
+        container: ${{ variables.ubuntu2204Container }}
         excludeOmniSharpTests: true        # ✅
```

Comment:
This flag should only be set for source-only builds.

Reply:
Fixed